test/local_tests/test_with_mock_workspace.py

```diff
@@ -74,7 +75,7 @@ def test_noproject(self):
 
         out = self.cmake(CMAKE_PREFIX_PATH=self.installdir,
                          expect=fail)
-        print("failed as expected, out=", out)
+        print(("failed as expected, out=", out))
```

Comment:
The extra parenthesis don't seem to make any sense. Why are they being introduced?